### PR TITLE
Add workflow auto-tooting on pipeline release

### PR DESCRIPTION
**Pull request by jfy133**
Attempt at adding an auto-toot on pipeline release inspired by @nevrome from https://github.com/poseidon-framework/poseidon-hs

We need someone with mastodon keys in @nf-core/core  to get the ACCESS_TOKEN key though (see [here](https://github.com/rzr/fediverse-action#usage) for docs on getting this).

Also this is not tested(!) so someone will need to do that who has a better idea on how to do it :grimacing: (maybe via `test-pipeline?`)

## PR checklist

- [ ] This comment contains a description of changes (with reason)
- [ ] `CHANGELOG.md` is updated
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] Documentation in `docs` is updated

### CODE REVIEW

**Comment by jfy133** [comment]
Good idea! Like I said, I don't have the access to the secrets so I can't test this myself :\

File: nf_core/pipeline-template/.github/workflows/release-announcments.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+name: release-announcements
+
+on:
+  release:
+    types: [published]
```
Comment (jfy133) [inline nf_core/pipeline-template/.github/workflows/release-announcments.yml:5]:
```suggestion
    types: [published]
  workflow_dispatch:
```

**Comment by jfy133** [comment]
> I can edit the GHA secrets but don't have access to Mastodon. You can send me a private one that is secret for testing? Developing CICD is haaarrrdddd.

Ah sorry: I don't have access to the Mastodon token _for_ the secrets - the GHA secrets options I do have permissions for :grimacing: so same situation as you are :sweat_smile:

**Review by mirpedrol — APPROVED** [review]
We have the secrets now 👏 so I think we are good to merge and test it! 🙂

**Comment by jfy133** [comment]
Feel free to merge!

**Review by mashehu — APPROVED** [review]

**Comment by mirpedrol** [comment]
Toot worked fine! 🥳 
Tweet doesn't seem to work yet, but it's probably due to keys, so I will merge this PR 🙂

**Pull request merged**